.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.16]
```

Comment:
if using matrix, then worth testing on other versions too. can also add a mix of OSs:

```
jobs:
  test:
    name: Test on node ${{ matrix.node }} and ${{ matrix.os }}

    runs-on: ${{ matrix.os }}

    strategy:
      matrix:
        node:
          - 8
          - 10
          - 12
        os:
          - ubuntu-latest
          # - windows-latest
          - macOS-latest
```

Reply:
Not a problem! 👍

Reply:
I can add other yml files for the different OS since it isn't just npm commands.

Reply:
Other versions would conflict with the engine we define in `package.json`
And the best practice for apps is to have one specific version.

Reply:
Okay, I'm pretty sure this is back to what I had at the start. But are you guys happy with this now?

Reply:
> And the best practice for apps is to have one specific version.

Right, but I don't think it's the best practice to dictate which version of Node the developer / production should run in. Sometimes you cannot control which Node version is installed. That is why it should be tested on all major Node versions.